src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -890,8 +901,6 @@ private void RegisterCustomNodeDefinitionWithEngine(CustomNodeDefinition definit
                 Workspaces.OfType<HomeWorkspaceModel>().SelectMany(ws => ws.Nodes),
```

Comment:
Yep, suggestion as in my previous comment, which is to name it `GenerateGraphSyncDataForCustomNode` :)